aiohttp/helpers.py

```diff
@@ -106,6 +106,8 @@ def all_tasks(
 
 coroutines._DEBUG = old_debug  # type: ignore
 
+json_re = re.compile(r'^application/(?:[\w.+-]+?\+)?json')
```

Comment:
> `+?`
This is probably not what you want.

Reply:
This regexp pattern is used by client code for years.
What is wrong with it?

Reply:
yes, sorry for not pointing out to the fact that we simply moved `json_re` from this place: https://github.com/aio-libs/aiohttp/pull/3889/files#diff-2fa8ab44317df4fbb7864536a6794d7aL83

Reply:
@asvetlov `+` means one or more, `?` means `0` or `1`. I don't get this syntax. Does it essentially mean `*` (`0` or more times)?

Reply:
Oh, that's lazy mode. Nevermind.